### DISCUSSION TITLE
Is it possible to use new bootstrap zip file for android-5 branch?

### DISCUSSION THREAD

**Discussion by Biswa96** [termux/termux-app#2667] [Q&A] [answered]
If I use the APK file from android-5 branch any apt/pkg command fails with security related error. Is it possible to use new boostrap zip file from https://github.com/termux/termux-packages/releases/ ?

**Comment by agnostic-apollo** [comment, marked as answer]
New releases are only compatible with Android `7+`.

Check https://github.com/termux/termux-app#installation
